react/utils/index.tsx

```diff
@@ -83,6 +89,9 @@ export function generateConfigs(schema: any, intl: IntlShape, mobile: boolean) {
       case 'email':
         return { width: widths.email, cellRenderer: EmailFieldCell }
 
+      case 'gender':
+        return {cellRenderer: GenderFieldCell}
```

Comment:
I think the `width` is necessary here to mantain the default behavior

```suggestion
        return { width: widths.default, cellRenderer: GenderFieldCell }
```

Reply:
thanks! done!